core/src/main/java/utilcalc/core/parser/Parser.java

```diff
@@ -0,0 +1,62 @@
+package utilcalc.core.parser;
+
+import static utilcalc.core.parser.Parser.GeneralSection.*;
+import static utilcalc.core.parser.ParserUtils.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.tomlj.*;
+import utilcalc.core.model.input.ReportInputs;
+
+public final class Parser {
+
+    static final class GeneralSection {
```

Comment:
Porad si myslim, ze je ta trida zbytecna. Uz to, zes ji musel dat non-private jen kvuli tomu importu naznacuje, ze je neco spatne. Ty konstanty jsou pouzivane jen v parseru, mely by tedy byt privatni.